lms/djangoapps/grades/models.py

```diff
@@ -363,3 +365,75 @@ def _prepare_params_visible_blocks_id(cls, params):
         """
         params['visible_blocks_id'] = params['visible_blocks'].hash_value
         del params['visible_blocks']
+
+
+class PersistentCourseGrade(TimeStampedModel):
+    """
+    A django model tracking persistent course grades.
+    """
+
+    class Meta(object):
+        # Indices:
+        # (course_id, user_id) for individual grades
+        # (course_id) for instructors to see all course grades
```

Comment:
Nit: Since we specify that the `user_id` index is explicitly created, could we also add a symmetrical comment that the `course_id` index is implicitly created due to `unique_together`?